manager/logbroker/broker.go

```diff
@@ -41,11 +41,32 @@ func New() *LogBroker {
 	}
 }
 
+// Run the log broker
+func (lb *LogBroker) Run(ctx context.Context) error {
+	lb.mu.Lock()
+	lb.stopped = make(chan struct{})
+	lb.logQueue = watch.NewQueue()
+	lb.subscriptionQueue = watch.NewQueue()
+	lb.mu.Unlock()
+
+	select {
+	case <-lb.stopped:
+		return nil
+	case <-ctx.Done():
+		return ctx.Err()
+	}
```

Comment:
If `LogBroker` doesn't have a main loop, maybe it doesn't make sense to have a blocking `Run` function that only waits for `Stop` to be called. The manager could just call `New` whenever it becomes the leader. But I can see the argument for consistency. On the other hand, I dislike goroutines that do nothing.